.github/workflows/generator.yml

```diff
@@ -2,14 +2,13 @@ name: generator
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
 
   workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, 'ci skip')"
```

Comment:
Not necessary: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs